src/site/content/en/handbook/audience/index.md

```diff
@@ -55,17 +56,15 @@ Jaimie first took a web development class at their local community college and i
 * **Goal:** Learn about the most recent web development tools and best practices to create a personal site from scratch to apply for industry jobs.
 
 ## Writing to the audience
-All posts and codelabs should be targeted to one of our personas (or more, if their needs overlap).
-
-That said, we want our content to address as much of our audience as we can. How do you do that in a given piece, especially when the needs of different readers compete? One way to address multiple audience segments is to provide multiple paths for a given topic. For example, you might write:
+When you're writing a post or codelab, keep the web.dev personas in mind (or more, if their needs overlap). How do you do that in a given piece, especially when the needs of different readers compete? One way to address multiple audience segments is to provide multiple paths for a given topic. For example, you might write:
```

Comment:
This changes the meaning. The intent was to say (a) target a specific persona, but (b) also be mindful of how to serve audience members not represented by that persona. Kate's doing a larger overhaul of this page, so let's hold off on further fiddling until she has a draft :)